continuationsPluginExample/src/main/scala/examples/NonCompanionObjectContinuations.scala

```diff
@@ -0,0 +1,41 @@
+//package examples
```

Comment:
Should compile now with the updates to main to delegate to scalac via coursier launch.

Reply:
You can uncomment this.

Reply:
On main, there should be a `bypass-zinc` project that aggregates the projects that need raw scalac compilation, and changes to the ci alias that keep it from crashing zinc in ci: https://github.com/47deg/TBD/blob/56cf64c3027716558597f2c283572a6f4f9f2d5f/build.sbt#L17-L42

Reply:
thanks for flagging this.
done in [77b9be8](https://github.com/47deg/TBD/pull/81/commits/77b9be8fdbbd5dfd4ba461589c6b9cae4acc0367)